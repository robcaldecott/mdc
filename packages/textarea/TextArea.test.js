import React from "react";
import { TextArea } from ".";
import renderer from "react-test-renderer";

it("renders", () => {
  expect(
    renderer.create(
      <TextArea
        id="textarea"
        className="custom"
        value="Value"
        label="Label"
        fullWidth
        dense
        disabled
        rows="8"
        cols="40"
      />
    )
  ).toMatchSnapshot();
});
