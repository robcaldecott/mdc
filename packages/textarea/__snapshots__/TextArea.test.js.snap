// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="mdc-text-field mdc-text-field--textarea mdc-text-field--upgraded mdc-text-field--fullwidth mdc-text-field--disabled mdc-text-field--dense custom"
>
  <textarea
    className="mdc-text-field__input"
    cols="40"
    disabled={true}
    id="textarea"
    onBlur={[Function]}
    onFocus={[Function]}
    rows="8"
    value="Value"
  />
  <label
    className="mdc-text-field__label mdc-text-field__label--float-above"
    htmlFor="textarea"
  >
    Label
  </label>
</div>
`;
